hlx_statics/scripts/lib-adobeio.js

```diff
@@ -193,13 +195,46 @@ export function decorateInlineCodes(element) {
  * @param {*} container The container to inspect
  */
 export function buildCodes(container) {
-  const codes = [...container.querySelectorAll('main > div > pre > code')];
+  const codes = [...container.querySelectorAll('main > div pre')];
+
   codes.forEach((code) => {
+
+    const parentDiv = code.closest('div');
+    parentDiv.classList.add('code-container', 'codeblock-container');
```

Comment:
Will this work instead?

```suggestion
    parentDiv.classList.add('code-container');
```

I don't think this needs to be a `codeblock-container` (i.e. block with tabs). `code-container` makes sense since on line 204 below, we're building a `code` block, but not a `codeblock` block.